packages/modules/vehicles/vwid/libvwid.py

```diff
@@ -21,7 +21,7 @@ class vwid:
     def __init__(self, session):
         self.session = session
         self.headers = {}
-        self.log = logging.getLogger(__name__)
+        self.log = logging.getLogger("soc."+__name__)
```

Comment:
```suggestion
        self.log = logging.getLogger(__name__)
```
Ich habe das logging in 2.0 überarbeitet.